src/lib/components/ControlPad.svelte

```diff
@@ -123,7 +123,7 @@
 	{:else}
 		<button
 			aria-label="switch innings button"
-			class="rounded-xl py-2 w-full transition-colors {game.currentPlayer.color}"
+			class="rounded-xl py-2 w-full transition-colors bg-{game.currentPlayer.color}-ball"
```

Comment:
https://tailwindcss.com/docs/content-configuration#configuring-source-paths:~:text=Tailwind%20CSS%20works,Tailwind%20class%20names.

Reply:
For this reason, tailwind will break if you pass around partial class names. You need to pass around full tailwind classes or else they won't be generated
```suggestion
			class="rounded-xl py-2 w-full transition-colors {game.currentPlayer.color}"
```